tests/core/bookings/test_api.py

```diff
@@ -207,6 +207,20 @@ def test_cancel_booking(self):
         email_data2 = mails_testing.outbox[1].sent_data
         assert email_data2["MJ-TemplateID"] == 780015  # to offerer
 
+    def test_cancel_booking_twice(self):
+        booking = factories.BookingFactory()
+        initial_qt = booking.stock.dnBookedQuantity
```

Comment:
```suggestion
        initial_quantity = booking.stock.dnBookedQuantity
```

J'ai mis du temps à comprendre ce qu'étais `qt`

Reply:
fixed